deployments/ansible/roles/collector/README.md

```diff
@@ -56,27 +74,35 @@ how to use the role in a playbook with minimal required configuration:
 - `splunk_otel_collector_config`: Splunk OTel Collector config YAML file. Can be set to 
   `/etc/otel/collector/gateway_config.yaml` to install the collector in gateway
   mode. (**default:** `/etc/otel/collector/agent_config.yaml`)
+  For Windows this is the destination path of the collector config file on the node. 
+  The SPLUNK_CONFIG environment variable will be set with this value for the collector service.
+  (**default:** `%ProgramData%\Splunk\OpenTelemetry Collector\agent_config.yaml`)
```

Comment:
Why the definition is different for Windows? It should have the same meaning. So we can just change the default value to the following:
**default:** `/etc/otel/collector/agent_config.yaml` on Linux, `%ProgramData%\Splunk\OpenTelemetry Collector\agent_config.yaml` on Windows